package.json

```diff
@@ -114,13 +114,13 @@
     "@types/mocha": "^10.0.6",
     "@types/node": "^20.x",
     "@types/vscode": "^1.83.1",
-    "@typescript-eslint/eslint-plugin": "^6.20.0",
-    "@typescript-eslint/parser": "^6.21.0",
+    "@typescript-eslint/eslint-plugin": "^8.18.0",
+    "@typescript-eslint/parser": "^8.18.0",
     "@vscode/test-cli": "0.0.4",
     "@vscode/test-electron": "^2.3.9",
-    "eslint": "^8.56.0",
+    "eslint": "^9.16.0",
```

Comment:
Careful with this one, eslint 9.x changed a whole whack load and we haven't even upgraded it elsewhere. I'd suggest keeping this one locked to 8.x until the ecosystem has had more time to adapt their plugins.... Though i guess we aren't using anything but @typescript-eslint/eslint-plugin so it should be doable. But we'd probably need to change config and some cli options.

Reply:
Ahhh, okay, yeah, there are definitely a lot of breaking changes. Let me downgrade that.